x-pack/plugins/ml/public/application/components/anomalies_table/links_menu.js

```diff
@@ -53,6 +53,51 @@ class LinksMenuUI extends Component {
     };
   }
 
+  openInDiscover = async () => {
+    const {
+      services: { share },
+    } = this.props.kibana;
+    const discoverLocator = share.url.locators.get('DISCOVER_APP_LOCATOR');
+    const job = mlJobService.getJob(this.props.anomaly.jobId);
+
+    const index = job.datafeed_config.indices[0];
+    const dataViewId = (await getDataViewIdFromName(index)) || index;
+
+    const record = this.props.anomaly.source;
+    const from = timeFormatter(record.timestamp); // e.g. 2016-02-08T16:00:00.000Z
```

Comment:
Could you please use [this hook](https://github.com/elastic/kibana/blob/e10ac3c56e6c0e5144dfbbd8fe8b9d56bc1e643d/x-pack/plugins/ml/public/application/contexts/kibana/use_field_formatter.ts#L21) instead? It utilizes the `fieldFormats` service.

Reply:
Updated here https://github.com/elastic/kibana/pull/120450/commits/e982410ce8d3231a42b0a3f40821be9a6b317043

Reply:
Great 👍 Please bear in mind that default date formatting in Kibana is not `2016-02-08T16:00:00.000Z`. You need to pass the required options as a second argument for `useFieldFormatter `

Reply:
Looks like there's a timezone issue with the fieldFormatter services. I reverted back to use the previous timeFormatter for now https://github.com/elastic/kibana/pull/120450/commits/d03dfd4aa2ff76d933a7ee09c3fd416dd3245dde while investigating this further. As the from/to here is not shown to the users (it's only for the url), I think it should be fine.